docs/game_sdk/Activities.md

```diff
@@ -340,6 +341,44 @@ Fires when a user accepts a game chat invite or receives confirmation from Askin
 | ---------- | ------ | ---------------------------------- |
 | joinSecret | string | the secret to join the user's game |
 
+
+###### Example
+
+```cs
+// Received when someone accepts a request to join or invite.
+// Use secrets to receive back the information needed to add the user to the group/party/match
+activityManager.OnActivityJoin += secret => {
+    Console.WriteLine("OnJoin {0}", secret);
+    lobbyManager.ConnectLobbyWithActivitySecret(secret, (Discord.Result result, ref Discord.Lobby lobby) =>
+    {
+        Console.WriteLine("Connected to lobby: {0}", lobby.Id);
+        // Connect to voice chat, used in this case to actually know in overlay if your successful in connecting.
+        lobbyManager.ConnectVoice(lobby.Id, (Discord.Result voiceResult) => {
+
+            if (voiceResult == Discord.Result.Ok)
+            {
+                Console.WriteLine("New User Connected to Voice! Say Hello! Result: {0}", voiceResult);
+            }
+            else
+            {
+                Console.WriteLine("Failed with Result: {0}", voiceResult);
+            };
+        });
+		//Connect to given lobby with lobby Id
+        lobbyManager.ConnectNetwork(lobby.Id);
+        lobbyManager.OpenNetworkChannel(lobby.Id, 0, true);
+        foreach (var user in lobbyManager.GetMemberUsers(lobby.Id))
+        {
+			//Send a hello message to everyone in the lobby
+            lobbyManager.SendNetworkMessage(lobby.Id, user.Id, 0,
+                Encoding.UTF8.GetBytes(String.Format("Hello, {0}!", user.Username)));
+        }
+		//Sends this off to a Activity callback named here as 'UpdateActivity' passing in the discord instance details and lobby details
+        UpdateActivity(discord, lobby);
```

Comment:
Not sure what the `discord` object is here?

Reply:
Oh gotcha, I see. You're passing the core into another function called `UpdateActivity()` and then the lobby object so that you can update the presence.

I think without that function actually here, it's a little confusing. I'd say either omit this line, or make a small example here of what that function does.

Reply:
Added in UpdateActivity Example to that.

Reply:
Didn't include joinActivitySecret though, since that gets set outside the scope of UpdateActivity mind you.